src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-console */
+import classNames from 'classnames';
+import React from 'react';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  onSetFilterGlobal: React.Dispatch<React.SetStateAction<string>>
+  selectedFilter: string
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onSetFilterGlobal,
+  selectedFilter,
+}) => {
+  console.log('Filetr LINK', selectedFilter);
+
+  const handleAnchorClick = (
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
```

Comment:
```suggestion
    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
```

Reply:
nope) its common practice to use 'e' for events (every link has the same))

Reply:
i suggest using `event`